backend/tests/controllers/test_executions/test_start_test.py

```diff
@@ -29,15 +33,25 @@
 )
 from test_observer.data_access.models_enums import (
     FamilyName,
+    TestExecutionReviewDecision,
     TestExecutionStatus,
 )
 from tests.data_generator import DataGenerator
 
+Execute: TypeAlias = Callable[[dict[str, Any]], Response]
 
-def test_creates_all_data_models(db_session: Session, test_client: TestClient):
-    response = test_client.put(
-        "/v1/test-executions/start-test",
-        json={
+
+@pytest.fixture
+def execute(test_client: TestClient) -> Execute:
```

Comment:
`execute` seems to be a bit too generic for this helper method. Perhaps we can rename it to `start_test_execution` or something else that will indicate that this method makes a `PUT` request to start the test execution?

Reply:
I opted for the generic name because this whole file is called `test_start_test.py` so by definition execute would just start the test